Dockerfile.py3

```diff
@@ -122,8 +122,8 @@ RUN useradd --create-home --shell /bin/bash ${USER}
 RUN usermod -append --groups sudo ${USER}
 RUN echo "%sudo ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
 
-
-RUN pip3 install --upgrade cython==0.28.6
+# install cython for python 2 (for python 3 it's inside the venv)
+pip2 install --upgrade Cython==0.28.6
```

Comment:
You're missing the `RUN` here:
```
Error response from daemon: Dockerfile parse error line 126: unknown instruction: PIP2
```
ahah don't give up :sweat_smile:

Reply:
> ahah don't give up :sweat_smile:

I won't I'm stubborn :joy: